Commands Rewrite

Quantiset opened the pull request:
Following #26, I thought I could chip in and sorta update it to the new branch. Now, it does need a bit of testing (I tested most commands other than the memes and bobross command) but I hope it will update the bot a bit :).

Quantiset committed:
Update main.py for the commands rewrite

Following #26, I thought I could chip in and sorta update it to the new branch. Now, it does need a bit of testing (I tested most commands other than the memes and bobross command) but I hope it will update the bot a bit :).

Quantiset committed:
Update main.py

winston-yallow commented on main.py:
```diff
@@ -351,72 +352,60 @@ async def on_ready() -> None:
 
-@client.event
-async def on_message(message: Any) -> None:
+@client.command()
```
Just a small note: This will probably eliminate the aliases `!ross` and `!br`. I think you can pass aliases into the command decorator like this:
```python
@client.command(aliases=['ross', 'br'])
```

Quantiset committed:
Should fix the alias issues

Quantiset committed:
changed !code command to better represent code

Quantiset committed:
fixed ANOTHER problem in the !code function

Quantiset committed:
Accidental paste

I am so sorry for continually adding commits; it is my first time properly using Github so please excuse me
